server/app.js

```diff
@@ -78,7 +84,7 @@ function login(socket, msg){ // user sends a log in request and the server check
         return;
     }
     else{// send the current table state
-        socket.emit('tableState', JSON.stringify(tables[0].getTableState()));
+        //socket.emit('tableState', JSON.stringify(tables[0].getTableState()));
```

Comment:
Is this still being implemented?

Reply:
no, i realized that the user would not be sent the table state on login, they first would see the list of tables. one thing we really need to get done is linking up all the pages together so i can start using real authentication and real profiles. we should address this in tomorrows meeting